### PR TITLE
Set body colour in darkmode

### PR DESCRIPTION
**Pull request by abeddow91**
## What does this change?
Sets the dark mode body colour in the global styles.

## Why?
We use source's css resets to define our body colour. We need to reset the reset when the user is in dark mode so that body text displays correctly. By co-locating this with the dark mode palette, we don't need to add any further checks elsewhere to see if the user is in darkmode. 

## Screenshots

| Before      | After      |
| ----------- | ---------- |
| ![before][] | ![after][] |

[before]: https://github.com/guardian/dotcom-rendering/assets/20416599/001433c2-95cf-4cc0-8055-43e423ba21e6
[after]: https://github.com/guardian/dotcom-rendering/assets/20416599/51ddd794-cbe0-4d6b-9f56-fa3aeee736a3

### CODE REVIEW

**Comment by abeddow91** [comment]
Thanks @sophie-macmillan - yep this is the wrong colour! not sure how that happened. Looks like apps are using neutral.86 so i've plumped for that

**Review by mxdvl — APPROVED** [review]
I have to say that I do not love that the light and dark scheme `body` font colours are [defined in different files accross repositories](https://github.com/guardian/csnx/blob/6d553cd261caf5097b2b39d7922a167d81cd4e64/libs/%40guardian/source-foundations/src/utils/resets.ts#L53), but appreciate this need not be addressed in this PR specifically.

What about setting the `body { color: ${palette.neutral[7]}; }` explicitly in DCR?

**Comment by abeddow91** [comment]
@mxdvl yeah I did wonder about this. I wasn't sure if it was more confusing to have it set both in the css reset and then again in the global styles but perhaps its nicer as it co-locates the light and dark mode colours. I'll update this pr with this change.

File: dotcom-rendering/src/components/ArticlePage.tsx [hunk 1]
```diff
@@ -62,12 +67,18 @@ export const ArticlePage = (props: WebProps | AppProps) => {
 					:root {
 						/* Light palette is default on all platforms */
 						${paletteDeclarations(format, 'light')}
-
+						body {
+							color: ${sourcePalette.neutral[7]};
+						}
```
Comment (abeddow91) [inline dotcom-rendering/src/components/ArticlePage.tsx:72]: This is already defined in the css resets provided by libs. However, we're adding this here to make the light / dark coupling more explicit. @mxdvl

**Pull request merged**